mdoc/src/main/scala/mdoc/internal/cli/MainOps.scala

```diff
@@ -121,7 +121,14 @@ final class MainOps(
   }
 
   def generateCompleteSite(): Exit = {
-    val files = IO.inputFiles(settings)
+    val inIsFile = settings.in.toFile.isFile
+    val files = if (inIsFile) {
+      settings
+        .toInputFile(settings.in)
+        .fold(List.empty: List[InputFile])(List(_))
```

Comment:
Is this the same? Alternatively, I think pattern matching is more readable than using `Option.fold`
```suggestion
        settings.toInputFile(settings.in).toList
```

Reply:
Ahh, yea it is. This is much nicer.